package.json

```diff
@@ -34,9 +34,9 @@
     "ocis-kill": "killall ocis",
     "docker-kill": "docker kill phoenix-tests-selenium phoenix-tests-redis",
     "testing-app": "if [ -d tests/testing-app ]; then (cd tests/testing-app && git pull && pwd); else git clone --depth 1 https://github.com/owncloud/testing.git tests/testing-app; fi;",
-    "build-ocis": "if [ -d tests/ocis ]; then (cd tests/ocis && git reset --hard master && git pull); else git clone --depth 1 https://github.com/owncloud/ocis.git tests/ocis; fi; cd tests/ocis; make clean; make",
-    "ocis": "yarn run build-ocis; REVA_STORAGE_OWNCLOUD_REDIS_ADDR='localhost:6379' REVA_STORAGE_HOME_EXPOSE_DATA_SERVER=1 REVA_STORAGE_OC_EXPOSE_DATA_SERVER=1 PHOENIX_ASSET_PATH='./dist' tests/ocis/bin/ocis server &",
-    "ocis:mac": "yarn run build-ocis; REVA_STORAGE_OWNCLOUD_REDIS_ADDR='localhost:6379' PROXY_OIDC_ISSUER='https://host.docker.internal:9200' KONNECTD_INSECURE='true' KONNECTD_IDENTIFIER_REGISTRATION_CONF='./tests/acceptance/mac-identifier-registration.yml' KONNECTD_ISS='https://host.docker.internal:9200' KONNECTD_TLS='true' PHOENIX_WEB_CONFIG='./tests/acceptance/ocis-mac-config.json' PHOENIX_ASSET_PATH='./dist' bin/ocis server &",
+    "build-ocis": "if [ -d tests/ocis ]; then (cd tests/ocis && git reset --hard master && git pull); else git clone --depth 1 https://github.com/owncloud/ocis.git tests/ocis; fi; cd tests/ocis/ocis; make clean; make",
+    "ocis": "yarn run build-ocis; STORAGE_HOME_DRIVER=owncloud STORAGE_USERS_DRIVER=owncloud REVA_STORAGE_OWNCLOUD_REDIS_ADDR='localhost:6379' REVA_STORAGE_HOME_EXPOSE_DATA_SERVER=1 REVA_STORAGE_OC_EXPOSE_DATA_SERVER=1 PHOENIX_ASSET_PATH='./dist' tests/ocis/ocis/bin/ocis server &",
+    "ocis:mac": "yarn run build-ocis; STORAGE_HOME_DRIVER=owncloud STORAGE_USERS_DRIVER=owncloud REVA_STORAGE_OWNCLOUD_REDIS_ADDR='localhost:6379' PROXY_OIDC_ISSUER='https://host.docker.internal:9200' KONNECTD_INSECURE='true' KONNECTD_IDENTIFIER_REGISTRATION_CONF='./tests/acceptance/mac-identifier-registration.yml' KONNECTD_ISS='https://host.docker.internal:9200' KONNECTD_TLS='true' PHOENIX_WEB_CONFIG='./tests/acceptance/ocis-mac-config.json' PHOENIX_ASSET_PATH='./dist' bin/ocis/ocis server &",
```

Comment:
Is the `REVA_STORAGE_REDIS_ADDR` needed? I think most reva vars have been removed from this command, right? Also, if it's `localhost` then it probably never should have work in this command.

Reply:
you are right, and the name of the variable would be anyway wrong, see https://github.com/owncloud/ocis/blob/17a180e3396fad9e6136e40b62859204e0033d51/storage/pkg/flagset/driverowncloud.go#L43
I've deleted all mentioning of `REVA_STORAGE_REDIS_ADDR`